src/server/pps/server/worker_rc.go

```diff
@@ -96,17 +96,36 @@ func (a *apiServer) workerPodSpec(options *workerOptions, pipelineInfo *pps.Pipe
 		Name:  "STORAGE_BACKEND",
 		Value: a.storageBackend,
 	}, {
-		Name:  "PORT",
-		Value: strconv.FormatUint(uint64(a.port), 10),
+		Name:  "POSTGRES_USER",
+		Value: a.env.Config().PostgresUser,
 	}, {
-		Name:  "PEER_PORT",
+		Name:  "POSTGRES_PASSWORD",
```

Comment:
NB: both the worker and the sidecar need postgres access, I guess? so all of these env vars have to be set in both.

Reply:
Ah! Nice catch. Common Env Vars is nice addition

Reply:
It came out to -1 LOC

![](https://i.kym-cdn.com/photos/images/newsfeed/001/168/676/09e.gif)